app/editor/src/features/admin/users/UserForm.tsx

```diff
@@ -40,7 +41,7 @@ export const UserForm: React.FC = () => {
     lookups.roles.map((r) => new OptionItem(r.name, r.id)),
   );
 
-  const isLinkedToKeycloak = user.key !== '00000000-0000-0000-0000-000000000000';
+  const isAdminUser = isAdmin(user);
```

Comment:
You don't need this.  The admin is the only user who should be able to get to this form.

Reply:
Figured that may be the case... will get rid of